superset/daos/datasource.py

```diff
@@ -45,15 +44,14 @@ class DatasourceDAO(BaseDAO[Datasource]):
     @classmethod
     def get_datasource(
         cls,
-        session: Session,
```

Comment:
It is not apparent to me why the `session` was included here. There are unit tests which use a session other than `db.session` but, per [here](https://github.com/apache/superset/blob/master/tests/unit_tests/conftest.py#L65), the global `db.session` is mocked to use said session.